src/index.html

```diff
@@ -5,4 +5,151 @@
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fballet.png?v=1605143408910" alt="Ballet">
+            <h2>Ballet</h2>
+        </li>
+        <li class="pointe">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fpointe.png?v=1605143399024" alt="Pointe">
+            <h2>Pointe</h2>
+        </li>
+        <li class="modern">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fmodern.png?v=1605143401181" alt="Modern">
+            <h2>Modern</h2>
+        </li>
+        <li class="tap">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Ftap.png?v=1605143407247" alt="Tap">
+            <h2>Tap</h2>
+        </li>
+        <li class="jazz">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fjazz.png?v=1605143407596" alt="Jazz">
+            <h2>Jazz</h2>
+        </li>
+        <li class="hiphop">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fhiphop.png?v=1605143407749" alt="Hip-Hop">
+            <h2>Hip-Hop</h2>
+        </li>
+        <li class="african">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fafrican.png?v=1605143399790" alt="African">
+            <h2>African</h2>
+        </li>
+        <li class="little-ones">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Flittleones.png?v=1605143402975" alt="little-ones">
+            <h2>Little Ones</h2>
+        </li>
+        <li class="repertory">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Frepertory.png?v=1605143401041" alt="repertory">
+            <h2>Repertory</h2>
+        </li>
+        <li class="adult-classes">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fadultclasses.png?v=1605143403559" alt="dult Classes">
+            <h2>Adult Classes</h2>
+        </li>
+    </ul>
+</div>
+<div class="Register">
+    <h1>2020 Fall Registration</h1>
+    <button class="accordion">Step 1</button>
+    <div class="panel">
+        <p>
+            CURRENT STUDENTS: Check your email! You will have class placements sent to you. If you want to add a class you have not taken before, please contact us before registering. <br>
+            PROSPECTIVE STUDENTS: Read our class descriptions, then call or email us to discuss appropriate class placement.
+        </p>
+    </div>
+    <button class="accordion">Step 2</button>
+    <div class="panel">
+        <p>
+            REVIEW CALENDAR, TUITION, REGISTRATION PROCEDURES<br>
+            Double ​check the classes you would like to take. Some classes meet twice per week. Make note of any potential conflicts and talk to us if you have one.
+            Commit to the year or join us for our 13-Week Session! Save money by paying in full. You will otherwise be put on a monthly payment plan.</p>
```

Comment:
Same note about using multiple `p` elements. Also, here and above, consider using another heading level or a `span` to achieve the all caps text (see [HTML best practices](https://mica-web.github.io/learn/html/best-practices#do-not-use-all-caps-in-your-markup-content-for-presentational-purposes)).